src/edu/umass/ckc/wo/ttmain/ttservice/classservice/impl/TTReportServiceImpl.java

```diff
@@ -1,1296 +1,1302 @@
-            perProblemReportBeanObj.setPercStudentsGaveUp((int)Math.round(100.0 / perProblemReportBeanObj.getNoStudentsSeenProblem() * perProblemReportBeanObj.getPercStudentsGaveUp()));
-            perProblemReportBeanObj.setGetPercStudentsSolvedEventually((int)Math.round(100.0 / perProblemReportBeanObj.getNoStudentsSeenProblem() * perProblemReportBeanObj.getGetPercStudentsSolvedEventually()));
-            perProblemReportBeanObj.setGetGetPercStudentsSolvedFirstTry((int)Math.round(100.0 / perProblemReportBeanObj.getNoStudentsSeenProblem() * perProblemReportBeanObj.getGetGetPercStudentsSolvedFirstTry()));
-
-        });
-        return perProblemReportBeanMap;
-    }*/
-
-
-    private Map<String, PerProblemReportBean> generatePerProblemReportForGivenProblemID(String classId, List<String> problemIdsList, Map<String, String> problemDescriptionMap){
-        Map<String, PerProblemReportBean> perProblemReportBeanMap = new LinkedHashMap<String, PerProblemReportBean>();
-        Map<String, String> selectParams = new LinkedHashMap<String, String>();
-        String URI = Settings.probPreviewerPath;
-        String html5ProblemURI = Settings.html5ProblemURI;
-        selectParams.put("classId", classId);
-        for (String problemId : problemIdsList) {
-            selectParams.put("problemId", problemId);
-            PerProblemReportBean perProblemReportBean = namedParameterJdbcTemplate.query(TTUtil.PER_PROBLEM_QUERY_FIFTH, selectParams, new ResultSetExtractor<PerProblemReportBean>() {
-                @Override
-                public PerProblemReportBean extractData(ResultSet resultSet) throws SQLException, DataAccessException {
-                    PerProblemReportBean perProblemReportBean = new PerProblemReportBean();
-                    while (resultSet.next()) {
-                        perProblemReportBean.setNoStudentsSeenProblem(resultSet.getInt("noOfStudents"));
-                    }
-                    Problem probDetails = ProblemMgr.getProblem(Integer.valueOf(problemId));
-                    if (probDetails != null) {
-                        if ("flash".equals(probDetails.getType())) {
-                            perProblemReportBean.setProblemURLWindow(URI + "?questionNum=" + probDetails.getProbNumber());
-                        } else {
-                            perProblemReportBean.setProblemURLWindow(html5ProblemURI + probDetails.getHTMLDir() + "/" + probDetails.getResource());
-                        }
-                    } else {
-                        perProblemReportBean.setProblemURLWindow(html5ProblemURI);
-                    }
-
-                    return perProblemReportBean;
-                }
-            });
-            perProblemReportBean.setSimilarproblems("Similar Problems");
-            String[] problemDescriptionValues = problemDescriptionMap.get(problemId).split("~~");
-            p
```

Comment:
Do we have any real changes for this file? As I do not see any changes as such but the whole file is coming in the changed file list

Reply:
This file got converted to unix eol format. I have converted it back and will replace theentire contents.  The were no substantive changes,